hnn_core/network.py

```diff
@@ -651,6 +651,9 @@ def add_poisson_drive(self, name, *, tstart=0, tstop=None, rate_constant,
                                  f"n_drive_cells='n_cells'. Got cell_specific"
                                  f" cell_specific={cell_specific} and "
                                  f"n_drive_cells={n_drive_cells}.")
+        elif isinstance(rate_constant, float):
+            rate_constant = {cell_type: rate_constant for cell_type in
+                             target_populations}
```

Comment:
smoke test for this?

Reply:
okay got it. Maybe a small comment next to `simulate dipole` explaining that this is tested would be good.